docs_src/V3TopLevelMigration.md

```diff
@@ -106,11 +111,39 @@ The field that has changed in V3 is the `apiVersion` which is now set to `v3`.
 
 Most of the data stored by Core Metadata will be recreated when the V3 versions of the Device Services start-up. The statically declared devices will automatically be created and device discovery will find and add existing devices. Any device profiles, devices, provision watchers created manually via the V2 REST APIs will have to be recreated using the V3 REST API. Any manually-applied `AdministrativeState` settings will also need to be re-applied.
 
+#### API Change
+- Add/ Update/ Get device
+    - Remove `LastConnected`, `LastReported` and `UpdateLastConnected` from device model
+    - Updated ProtocolProperties to have typed value
+
+- Add/ Update/ Get deviceprofile
+    - Added `optional` field in ResourceProperties
+    - Updated the data type of `mask`, `shift`, `scale`, `base`, `maximum` and `minimum` from `string` to `number` in ResourceProperties
+
+- Get UOM 
+    - Changed the response format from TOML to YAML
+
+- Add/ Update ProvisionWatcher
+    - Allowed empty string profile name when adding or updating the ProvisionWatcher
```

Comment:
There are additional changes to the ProvisionWatcher DTO that need to be here. `DiscoveredDevice` device was added, so custom provision watchers need to be migrated to the new DTO.
https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/dtos/provisionwatcher.go#L21